api/CSS.json

```
@@ -1635,13 +1635,13 @@
               "version_added": "12.1"
             },
             "safari": {
-              "version_added": false
+              "version_added": "9"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "9"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4"
```

This should be `"4.0"`:

```suggestion
              "version_added": "4.0"
```